From ad63f68a37fe341ccd1bd66519ec836df2751814 Mon Sep 17 00:00:00 2001
From: Raghava Chowdam <rchowdam@codeaurora.org>
Date: Mon, 6 Jan 2020 19:12:46 +0530
Subject: [PATCH] meta-qti-eap: Fixes for systemd messgaes.

Journal was disabled for better KPI and the errors generated by the
service files trying to connect to journal socket for logging resolved
through this patch.

The groups render and kvm fixes are applicable for VT similar to
SA415m through this patch.

Srv is a serve folder. It holds the site's specific data, which is not required.

Change-Id: I30a4582c06d8e66d7a277adbe0776b3f912e4cb9

diff --git a/src/core/execute.c b/src/core/execute.c
index 8ac69d1..7b060bc 100644
--- a/src/core/execute.c
+++ b/src/core/execute.c
@@ -644,7 +644,7 @@ static int setup_output(
         case EXEC_OUTPUT_JOURNAL_AND_CONSOLE:
                 r = connect_logger_as(unit, context, params, o, ident, fileno, uid, gid);
                 if (r < 0) {
-                        log_unit_warning_errno(unit, r, "Failed to connect %s to the journal socket, ignoring: %m", fileno == STDOUT_FILENO ? "stdout" : "stderr");
+
                         r = open_null_as(O_WRONLY, fileno);
                 } else {
                         struct stat st;

diff --git a/rules/50-udev-default.rules.in b/rules/50-udev-default.rules.in
index 191f56f..3ff26bd 100644
--- a/rules/50-udev-default.rules.in
+++ b/rules/50-udev-default.rules.in
@@ -36,8 +36,6 @@ SUBSYSTEM=="dvb", GROUP="video"
 SUBSYSTEM=="media", GROUP="video"
 SUBSYSTEM=="cec", GROUP="video"

-SUBSYSTEM=="drm", KERNEL=="renderD*", GROUP="render", MODE="@GROUP_RENDER_MODE@"
-SUBSYSTEM=="kfd", GROUP="render", MODE="@GROUP_RENDER_MODE@"

 SUBSYSTEM=="sound", GROUP="audio", \
   OPTIONS+="static_node=snd/seq", OPTIONS+="static_node=snd/timer"
@@ -78,9 +76,6 @@ KERNEL=="tun", MODE="0666", OPTIONS+="static_node=net/tun"

 KERNEL=="fuse", MODE="0666", OPTIONS+="static_node=fuse"

-# The static_node is required on s390x and ppc (they are using MODULE_ALIAS)
-KERNEL=="kvm", GROUP="kvm", MODE="@DEV_KVM_MODE@", OPTIONS+="static_node=kvm"
-
 SUBSYSTEM=="ptp", ATTR{clock_name}=="KVM virtual PTP", SYMLINK += "ptp_kvm"

 LABEL="default_end"

diff --git a/tmpfiles.d/home.conf b/tmpfiles.d/home.conf
index 9f25b83..dbffd4f 100644
--- a/tmpfiles.d/home.conf
+++ b/tmpfiles.d/home.conf
@@ -8,4 +8,4 @@
 # See tmpfiles.d(5) for details

 Q /home 0755 - - -
-q /srv 0755 - - -
+#q /srv 0755 - - -

